spark/v3.1/spark/src/main/java/com/netease/arctic/spark/ArcticSparkCatalog.java

```diff
@@ -206,15 +201,39 @@ public Table createTable(
     }
   }
 
-  private Schema checkAndConvertSchema(Schema schema, Map<String, String> properties) {
+  private Schema checkAndConvertSchema(StructType schema, Map<String, String> properties) {
+    Schema convertSchema;
+    boolean defaultTimeZoneUsed;
+    try {
+      CatalogMeta catalogMeta = client.getCatalog(catalog.name());
+      String catalogType = catalogMeta.getCatalogType();
+      SparkSession sparkSession = SparkSession.active();
+      if (CATALOG_TYPE_HIVE.equals(catalogType)) {
+        defaultTimeZoneUsed = true;
+      } else {
+        defaultTimeZoneUsed = Boolean.parseBoolean(
+            sparkSession.conf().get(USE_TIMESTAMP_WITHOUT_TIME_ZONE_IN_NEW_TABLES,
+                "false"));
```

Comment:
Maybe we can change the value in `com.netease.arctic.spark.SparkSQLProperties#USE_TIMESTAMP_WITHOUT_TIME_ZONE_IN_NEW_TABLES_DEFAULT` to `false`?